internal/cache/file/downloader/job.go

```diff
@@ -87,7 +87,8 @@ type Job struct {
 	// is responsibility of JobManager to pass this function.
 	removeJobCallback func()
 
-	mu locker.Locker
+	mu              locker.Locker
+	fileCacheConfig *config.FileCacheConfig
```

Comment:
move the fileCacheConfig just below fileSpec and in place of enableCrcCheck

Reply:
done